study.md

```diff
@@ -57,5 +57,5 @@ Starting with a client making a GET request to a particular URL, describe how
 the parts of Rails interact to produce and send a response.
 
 ```md
-<!-- your answer here -->
+The client makes a GET request from the browser. The web server chooses the correct route for that request and sends the information to the controller layer. The controller layer takes that info and makes it so the model layer can read it and process it. The model layer controls the request in the back end and if all goes well, spits the confirmed info back into the controller layer which processes and sends the appropriate response to the server and browser. 
```

Comment:
Maybe you’re just clarifying so I understand, but the question specifically asks for the process when a client makes a GET request. 

> On Feb 16, 2016, at 11:26 AM, Jeffrey Horn <notifications@github.com> wrote:
> 
> In study.md <https://github.com/ga-wdi-boston/rails-api-study/pull/11#discussion_r53035313>:
> 
> > @@ -57,5 +57,5 @@ Starting with a client making a GET request to a particular URL, describe how
> >  the parts of Rails interact to produce and send a response.
> >  
> >  ```md
> > -<!-- your answer here -->
> > +The client makes a GET request from the browser. The web server chooses the correct route for that request and sends the information to the controller layer. The controller layer takes that info and makes it so the model layer can read it and process it. The model layer controls the request in the back end and if all goes well, spits the confirmed info back into the controller layer which processes and sends the appropriate response to the server and browser. 
> Not every request is a GET request.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/ga-wdi-boston/rails-api-study/pull/11/files#r53035313>.
>